autoscheduler/autoscheduler/middleware/logging_middleware.py

```diff
@@ -24,3 +26,17 @@ def __call__(self, request):
         response = self.get_response(request)
 
         return response
+
+    def process_exception(self, request, _):
```

Comment:
Will this double the exception emails we get? E.g. if we get a 400 error (and it sends us an email), will this trigger an email as well?

Reply:
No, the emails for log-based errors are based on whether errors are encountered in 5-minute timeframes and since this should be logged pretty much simultaneously with the actual error itself the email alert should overlap 99% of the time